.circleci/deploy_to_kubernetes

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -eu
+
+ROOT=$(dirname "$0")
+ENVIRONMENT="$1"
+ENVIRONMENT_DIR="$ROOT/../kubernetes_deploy/$ENVIRONMENT"
+
+if ! [ -d $ENVIRONMENT_DIR ] ; then
+  echo "$ENVIRONMENT_DIR not found"
```

Comment:
Would we enter this branch if `$1` is empty (missing)? It feels like `$ROOT/../kubernetes_deploy/` would pass this test.

Reply:
Note the `-u` in `#!/bin/sh -eu`.

>“set -u” makes the script exit if a referenced variable is not declared. Pretty useful since an undeclared variable can be mistaken for an empty one.

So in this case, `$1` isn't declared.

Reply:
TIL 😍 

💭 In this case, what do you think about also printing a usage line when the argument is missing? Might be easier for the users.